src/dstack/_internal/utils/ssh.py

```diff
@@ -157,3 +157,26 @@ def rsa_pkey_from_str(private_string: str) -> PKey:
 def generate_public_key(private_key: PKey) -> str:
     public_key = f"{private_key.get_name()} {private_key.get_base64()}"
     return public_key
+
+
+def check_required_ssh_version():
+    try:
+        result = subprocess.run(["ssh", "-V"], capture_output=True, text=True)
+        # TODO(loghi) - windows returns ssh -v to stdout. By default check_required_ssh_version returns True
```

Comment:
Should we leave TODO for windows? Judging by the comment, it should be easy to add a check for it too.